### DISCUSSION TITLE
帮忙找一下代码那里有错误，

### DISCUSSION THREAD

**Discussion by yixianbaowang** [MR-XieXuan/MiraiTravel#9] [Q&A] [answered]
![1676423062023](https://user-images.githubusercontent.com/125167113/218898875-c357b2b5-d3fd-44f9-81b6-7852a2bc5d06.jpg)
为什么执行第一行注释的代码之后会一直发送消息.

![image](https://user-images.githubusercontent.com/125167113/218899030-2608f7d5-f315-45f0-a591-3d4de482b8ea.png)
后台日志
![image](https://user-images.githubusercontent.com/125167113/218899126-da5286a3-4400-4913-8bdd-b5d68718fb1e.png)

**Comment by MR-XieXuan** [comment, marked as answer]
reply_message中只能传入MessageChain或者字符串。不应该传入其他类型的一个数组。你传入的$webhook['sender']既不属于MessageChain也不属于字符串,所以发送消息会失败，而easyMirai的replyMessage有发送失败后重试机制，所以会尝试发送4次你的消息。无疑，都是失败的。 @yixianbaowang
